compiler/aarch64/codegen/OpBinary.cpp

```diff
@@ -476,5 +476,12 @@ const OMR::ARM64::InstOpCode::OpCodeBinaryEntry OMR::ARM64::InstOpCode::binaryEn
 		0x1E604800,	/* FMAX      	fmaxd	 */
 		0x1E205800,	/* FMIN      	fmins	 */
 		0x1E605800,	/* FMIN      	fmind	 */
+	/* Vector Move (2 source) */
```

Comment:
Same comment as that for OMRInstOpCodeEnum.hpp.

Reply:
Will do accordingly, thanks. Shall I add `/* Hint instructions */` before the `nop` instruction? Otherwise, it might seem like it falls under the Vector instructions.

Reply:
Yes, that is a good idea.  The comment for `nop` is in OMRInstOpCodeEnum.hpp, but is missing in OpBinary.cpp.

Reply:
Yes, thanks for confirming that. I'll push the updated version shortly after I make the tests.